src/components/SwapChecker.tsx

```diff
@@ -27,6 +28,7 @@ import { getApiUrl } from "../utils/helper";
 import type {
     ChainSwap,
     ReverseSwap,
+    SomeSwap,
```

Comment:
That really the best name?

Reply:
It's either submarine, reverse or chain swap. If you come up with a better name, PRs welcome